info/v1/container.go

```diff
@@ -848,6 +848,26 @@ type PerfStat struct {
 	Cpu int `json:"cpu"`
 }
 
+// See: https://www.kernel.org/doc/Documentation/x86/intel_rdt_ui.txt
+type MemoryBandwidthMonitoringStats struct {
```

Comment:
Exported type should have a descriptive comment starting with `MemoryBandwidthMonitoringStats` , see e.g. `PerfStat` type description